package.json

```diff
@@ -52,7 +52,7 @@
     "grunt-contrib-concat": "0.5.1",
     "grunt-contrib-watch": "0.6.1",
     "grunt-shell": "1.1.2",
-    "jasmine-node": "2.0.0",
+    "jasmine-node": "1.14.5",
```

Comment:
The version of `jasmine` used there is `2.0.0` https://github.com/mhevery/jasmine-node/tree/Jasmine2.0/lib/jasmine-node/jasmine
and we found that some things are not working well. 
We could use directly the core library but this `1.x` version works just fine ;)